src/protocols/ForeignToplevelWlr.cpp

```diff
@@ -347,9 +347,6 @@ CForeignToplevelWlrProtocol::CForeignToplevelWlrProtocol(const wl_interface* ifa
     static auto P3 = g_pHookSystem->hookDynamic("activeWindow", [this](void* self, SCallbackInfo& info, std::any data) {
         const auto PWINDOW = std::any_cast<PHLWINDOW>(data);
 
-        if (!windowValidForForeign(PWINDOW))
```

Comment:
this will break if we have a window that is invalid for foreign. Should probably be `!valid() && PWINDOW`

Reply:
done

Reply:
done